vocode/streaming/transcriber/azure_transcriber.py

```diff
@@ -47,7 +47,7 @@ def __init__(
         config = speechsdk.audio.AudioConfig(stream=self.push_stream)
 
         speech_config = speechsdk.SpeechConfig(
-            subscription=getenv("AZURE_SPEECH_KEY"),
+            subscription=(transcriber_config.speech_key or getenv("AZURE_SPEECH_KEY")),
             region=getenv("AZURE_SPEECH_REGION"),
```

Comment:
`AZURE_SPEECH_REGION` also need to be in config?

Reply:
these types of env vars are fine since they arent secrets